app/controllers/community_controller.rb

```diff
@@ -9,12 +9,17 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
+
+    @countries = User.pluck(:country)
     @users = User.ordered(params[:sort],params[:direction])
         .group('users.id').with_all_associations_joined
     @users = @users.with_assigned_roles if Time.now.utc > (current_season.starts_at || Date.new)
     @users = @users.with_role(params[:role]) if params[:role].present? && params[:role] != 'all'
     @users = @users.with_interest(params[:interest]) if params[:interest].present? && params[:interest] != 'all'
-    @users = Kaminari.paginate_array(@users.search(params[:search])).page(params[:page])
+    @users = @users.as_coach_availability if params[:availability].present?
```

Comment:
This @users assignment is getting very long. Could you move this to a private method?